README.md

```diff
@@ -387,11 +387,56 @@ Pathfinder has a monitoring API which can be enabled with the `--monitor-address
 
 ### Metrics
 
-`/metrics` provides a [Prometheus](https://prometheus.io/) metrics scrape endpoint. Currently only one type of metric is available:
-- `rpc_method_calls_total` counter, where the label key `method` should be used to point to a particular RPC method to retrieve that method's total call count, for example: `rpc_method_calls_total{method="starknet_getStateUpdate"}`.
-- `extpy_processes_launched_total` counter incremented each time python subprocess is launched
-- `extpy_processes_exited_total` counter with labels, incremented each time python subprocess exits normally
-- `extpy_processes_failed_total` counter, incremented each time python subprocess exits abnormally
+`/metrics` provides a [Prometheus](https://prometheus.io/) metrics scrape endpoint. Currently the following metrics are available:
+
+#### RPC related counters
+
+- `rpc_method_calls_total`,
+- `rpc_method_calls_failed_total`,
+
+You __must__ use the label key `method` to retrieve a counter for a particular RPC method, for example:
+```
+rpc_method_calls_total{method="starknet_getStateUpdate"}
+rpc_method_calls_failed_total{method="starknet_chainId"}
+```
+
+#### Python subprocess related counters
+
+- `extpy_processes_launched_total` incremented each time python subprocess is launched
+- `extpy_processes_exited_total` with labels, incremented each time python subprocess exits normally
+- `extpy_processes_failed_total` incremented each time python subprocess exits abnormally
```

Comment:
I think we've briefly discussed this on Slack today. I din't think `cairo` is a better name, since these metrics are ultimately just about how many Python subprocesses we have. We've agreed there to leave them as they are.